### DISCUSSION TITLE
Uyuni Containerized 2024.08 on OpenSuse MicroOS and MicroOS automated updates scheduling

### DISCUSSION THREAD

**Discussion by RenownTwiz** [uyuni-project/uyuni#9312] [Q&A]
I've done some initial poking around at the MicroOS container host after I noticed that the Uyuni container instance was rebooting and down after midnight on occasions when I wasn't expecting it to be down.  We have a specific patching schedule with multiple instances of Uyuni and any one instance of Uyuni is only in use for a 2 week period of time each month.  So we can easily find a time each month that makes sense for the MicroOS instance to take updates but I'm not sure I understand exactly how to get automated updates rescheduled correctly because what I've tried is:

# Set reboot manager time
rebootmgrctl set-window *-*-29 10:00:00 1h30m

#Set transaction-update timer 
systemctl edit transactional-update.timer
#And set:
OnCalendar=*-*-29 06:00:00

Is this the right track?  After setting it it seemed like transaction update was still running after midnight so I'm thinking I've missed something?  Anyone know exactly what the procedure is to set the MicroOS automated update time/window?

**Comment by RenownTwiz** [comment]
FYI - It was noted in Uyuni Community hours 2024-11-28 that one can simply disable this as well.  The suggested command in chat was:

systemctl --now disable transactional-update.timer

**Comment by RenownTwiz** [comment]
In my case we don't really want to disable it, just schedule it for once a month on a day that we don't have other patching going on.

**Comment by ppanon2022** [comment]
It was nice that this was also covered in the 2024.12 release notes.
